zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/exception/BackupException.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.zookeeper.server.backup.exception;
+
+/**
+ * Signals an error has occurred during the ZooKeeper backup/restore process.
+ * For example: failure to copy a file which maybe due to invalid path or source file does not exist
+ */
```

Comment:
I think this is still a bit too vague. Is this used for I/O or some sort of a logical error?

Reply:
I added some more description. Does it look better now?

Reply:
This is better. Thanks